src/transformState.js

```diff
@@ -62,7 +62,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    if (transforms[i].operation === 'addProperties') {
+      const propertiesKeys = Object.keys(transforms[i].properties);
+
+      for (let j = 0; j < propertiesKeys.length; j++) {
+        state[propertiesKeys[j]] = transforms[i].properties[propertiesKeys[j]];
+      }
```

Comment:
You can simplify your code here by removing the loop and using Object.assign for transforms[i].properties